### ISSUE TITLE
[BITV] 9.4.1.2/3.16 - Custom editing menubar should be implemented using role=toolbar and not role=menubar. Currently selected styles on selected text in the editing region ("div" with "contenteditable=true") are shown visually (light blue background) but they are not programmatically determinable via aria-selected attribute. aria-activedescendant is not used on toolbar styling elements. Also missing is appropriate scripted focus management for the style toolbar and for its drop-down submenus like headings, insert image, insert emoi). (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37113]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n16

</details>

- [ ] 9.4.1.2/10.16 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n16
